### PR TITLE
New chart class for system events with customizations

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/charts.dart [hunk 1]
```diff
@@ -162,3 +162,45 @@ class DailyTimeseriesLineChartView {
     chart.update(new chartjs.ChartUpdateProps(duration: 0));
   }
 }
+
+ class SystenEventsTimeseriesLineChartView extends DailyTimeseriesLineChartView {
```
Comment (marianamarasoiu) [inline webapp/lib/charts.dart:166]: typo: `Systen` -> `System`

**Review by marianamarasoiu — APPROVED** [review]
Thanks @ktkization !

File: webapp/lib/charts.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/charts.dart:166]: Oops....I'll remember to rename it in the next branch

Comment (marianamarasoiu) [reply webapp/lib/charts.dart:166]: Great, thanks, or you can just open a separate PR for it
(and sorry, this is my fault, I jumped the gun on merging this PR, I should have given you the chance to fix the typo here!)

**Pull request merged**